experimental/schemas/orchestrator.yml

```
@@ -0,0 +1,69 @@
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster.name
+      level: extended
+      type: keyword
+      description: >
+        Name of the cluster.
+
+    - name: cluster.url
+      level: extended
+      type: keyword
+      description: >
+        URL of the cluster.
```

Perhaps this description should be specific about being the URL of the API used to manage the cluster.
Something like this.
```suggestion
        URL of the API used to manage the cluster.
```

---

Thanks, great suggestion!

Small adjustments to improve descriptions are fine, as long as we're not significantly changing the intent of the field.